tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
Generally speaking, it seems to be some minor instructions, setting up barycentric interpolation coefficients. (there's some internal registers for this, it seems)